integrations/database_test.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
```

Comment:
The file could be moved to `models/db`.

Reply:
No, `models/db` only uses sqlite for database during test.